bolt-design-system/bolt
# Link text and icon alignment enhancement

## Jira

http://vjira2:8080/browse/BDS-842

## Summary

Allow text link and text link with an icon to be truly inline.

## Details

1. Added `display` prop to handle 2 different scenarios:
    - inline would make both text and icon to be truly inline and they are able to wrap nicely within a paragraph of text.
    - flex would make the icon hang to the side and the text would wrap independently.

![screen shot 2019-01-07 at 5 23 23 pm](https://user-images.githubusercontent.com/3027663/50796855-fb32f980-12a0-11e9-99b6-36a33532395d.png)

2. Added `valign` prop to handle the text and icon's vertical alignment when `display` is set to flex.

3. Updated docs to reflect new features.

## How to test

Run the branch locally and check to see if the props are displaying correctly in the demos.
